### PR TITLE
fx(error: testing with post request not working)

### PR DESCRIPTION
**Pull request by yobuir**
resolving issue on testing apis with post requests

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -46,24 +47,25 @@ const options = {
...
-        if(whiteList.indexOf(origin) !== -1 || !origin){
-            callback(null, true)
-        }else{
-            callback(new Error("You are not allowed to access this apis"));
-        }
-    },
-    optionsSuccessStatus:200
-}
+// const whiteList= ['http://127.0.0.1:5500','http://localhost:3000'];
+// const corsOptions = {
```
Comment (chanceDira) [inline server.js:56]: Can you remove these comment since you are not using them.

**Review by chanceDira — COMMENTED** [review]
Good job @yobuir 

just one thing.

File: server.js (hunk 1 above)
Comment (yobuir) [reply server.js:56]: i was studying about IP whitelisting, but after deployment i will add the real url

Comment (chanceDira) [reply server.js:56]: Okay,

**Pull request merged**